src/ctelnet.cpp

```diff
@@ -929,17 +929,199 @@ std::pair<QString, int> cTelnet::getConnectionInfo() const
     }
 }
 
+void cTelnet::processTelnetDo(const std::string& command)
```

Comment:
Unfortunately the argument to this method was given the same name as a member of the class - so I renamed the latter so that it wasn't being hidden by this argument... :stuck_out_tongue_winking_eye: